packages/ui/src/components/templates/Dashboard/Chart/PieChart/PieChart.tsx

```diff
@@ -52,11 +54,12 @@ const PieChart = ({
   sectionAutoFocus = true,
   radius = 90,
   innerRadius = 60,
-  focusOnPress = false,
+  focusOnPress = true,
```

Comment:
When `focusOnPress` is false label should not be displayed. Don't forget to remove `paddingTop`